general/administration/hardware-acceleration.md

```diff
@@ -151,10 +151,11 @@ Install linux-headers and run the following command.
 apt-get install linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
 ```
 
-Alternatively, run this command if you are on stretch for compatibility.
+Alternatively, run thoses commands for compatibility.
```

Comment:
```suggestion
Alternatively, run the following commands for compatibility.
```

What do they mean by compatibility? Maybe we can use a better word

Reply:
I did wrote the original PR for this but this wasn't in it.
I used the formulation "Alternatively, use backports"